docs/cli/cmdline/script/run.md

```diff
@@ -0,0 +1,51 @@
+---
+title: terramate script run - Command
+description: With the terramate
+
+prev:
+  text: 'Script List'
+  link: '/cli/cmdline/script/list'
+
+next:
+  text: 'Script Tree'
+  link: '/cli/cmdline/script/tree'
+---
+
+# Script Run
+
+**Note:** This is an experimental command that is likely subject to change in the future.
+
+The `script run SCRIPT-NAME` command will run a Terramate script over a set of stacks. The script will run on all stacks under the current working directory where:
+
+- the script is available (scripts follow the same inheritance rules as globals)
+- any filters match. `script run` currently supports `--changed` and `--tags` filters.
+
+## Usage
+
+`terramate experimental script run [options] NAME`
+
+## Examples
+
+Run a script called "deploy" on all stacks where it is availablei:
+
+```bash
+terramate experimental script run deploy
+```
+
+Run a script called "deploy" on all changed stacks where it is available:
+
+```bash
+terramate experimental script run --changed deploy
+```
+
+Do a dry run of running the deploy script:
+
+```bash
+terramate experimental script run --dry-run deploy
+```
+
+Run a script called "deploy" in a specific stack without recursing into subdirectories:
+
+```bash
+terramate -C path/to/stack experimental script --no-recursive run deploy
```

Comment:
```suggestion
terramate -C path/to/stack experimental script run --no-recursive deploy
```

no recursive should not be an option of script but of run. cc @i4ki

Reply:
yes, it's a `script run` option. It does not work if provided to the `script` part.

Reply:
https://github.com/terramate-io/terramate/blob/main/cmd/terramate/cli/cli.go#L233